### PR TITLE
Removing Azure Deployment Step from CircleCi- No Longer Needed

### PR DESCRIPTION
**Pull request by Will-Hyde**
Were no longer using Circle-CI for azure deployments, so I am commenting out this step

### CODE REVIEW

**Comment by RyanCross** [comment]
Update data store dep to 0.1.13

**Review by RyanCross — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -32,34 +32,35 @@ jobs:
...
-          at: ~/project
-      - azure-cli/install
-      - azure-cli/login-with-service-principal
-      - run:
-          name: Install Azure functions tools
-          command: sudo npm i -g azure-functions-core-tools@4 --unsafe-perm true
-      - run:
-          name: Deploy to Azure function App
-          command: func azure functionapp publish af-datastore-api-rinkeby-prod --typescript
+  # deploy-rinkeby:
```
Comment (KennyZeroTech) [inline .circleci/config.yml:35]: Why comment all of this out instead of remove it?

File: .circleci/config.yml [hunk 2]
```diff
@@ -99,20 +100,20 @@ workflows:
...
-              only: develop
-      - deploy-mainnet:
-          requires:
-            - build
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /^v.*/
+      # - deploy-rinkeby:
```
Comment (KennyZeroTech) [inline .circleci/config.yml:103]: Same commenting question as above

File: .circleci/config.yml (hunk 1 above)
Comment (Will-Hyde) [reply .circleci/config.yml:35]: Perhaps we may want it in the future 🤷

**Review by KennyZeroTech — APPROVED** [review]
Looks good

**Pull request merged**